xlive/H2MOD/Modules/EventHandler/EventHandler.h

```diff
@@ -0,0 +1,116 @@
+
+enum EventTypes
+{
+	player_leave,
+	player_join,
+	gamestate_change,
+};
+enum GameState : BYTE
```

Comment:
This is already defined as https://github.com/pnill/cartographer/blob/development/xlive/Blam/Common/Common.h#L50

Reply:
Please use that instead

Reply:
done